topics/admin/faqs/galaxy-update-22.05.md

```diff
@@ -95,12 +96,14 @@ Here is the recommended update procedure with ansible:
    - uchida.miniconda is replaced with galaxyproject.conda
    - usegalaxy_eu.systemd is no longer needed
    - `galaxy_user_name` is defined in all.yml in the latest git-gat
+   - the `galaxy_job_config` needs to have a handling - assign set to `db-skip-locked`
```

Comment:
```suggestion
   - the `galaxy_job_config` needs to have a database handling specified - assign set to `db-skip-locked`
```

Reply:
I don't know what is the good way to write it. It refers to:
https://github.com/hexylena/git-gat/blob/303ee546e35ff27cf8a4adfe5efbeb5282ddc0ee/group_vars/galaxyservers.yml#L34-L35

Reply:
I changed my suggestion slightly.